icu4j/main/core/src/test/resources/com/ibm/icu/dev/data/cldr/units/unitsTest.txt

```diff
@@ -151,6 +151,7 @@ mass	;	ton	;	kilogram	;	907.18474 * x	;	907184.7
 mass	;	tonne	;	kilogram	;	1,000 * x	;	1000000.0
 mass	;	earth-mass	;	kilogram	;	5,972,200,000,000,000,000,000,000 * x	;	5.9722E27
 mass	;	solar-mass	;	kilogram	;	1,988,470,000,000,000,000,000,000,000,000 * x	;	1.98847E33
+night-duration	;	night	;	night	;	1 * x	;	1,000.00
```

Comment:
This had been manually removed in a previous integration and then restored for C but not J, so restoring it here.